debug_toolbar/panels/sql/tracking.py

```diff
@@ -126,6 +127,11 @@ def _decode(self, param):
         if isinstance(param, dict):
             return {key: self._decode(value) for key, value in param.items()}
 
+        # Handle binary data (e.g., GeoDjango EWKB geometry data)
+        if isinstance(param, (bytes, bytearray)):
+            # Mark as binary data for later reconstruction
+            return {"__djdt_binary__": base64.b64encode(param).decode("ascii")}
```

Comment:
We should move this logic into a shared utility that way we don't have the magic value `"__djdt_binary__"` floating around the code-base. Plus it would be clearer how the logic gets paired together.

I'm also curious if we should be using this with the Store logic. If we do, then we could potentially ignore this and handle it within `DebugToolbarJSONEncoder` and then have `DebugToolbarJSONDecoder` as above. What do you think?

Reply:
The Store logic converts all the panels data into JSON. So this binary parameter would eventually be converted to JSON to be added to the Store. So if the Store handles binary data properly, it may remove the need for the SQL panel to deal with it itself. Does that make more sense?